README.md

```diff
@@ -10,7 +10,7 @@ Now you can use a simple and pure javascript lib to mask your input elements, wi
 Let's live a lightweight client-side world using micro libraries as VanillaMasker is!
 Don't worry about where this will work, because this is a cross-browser and cross-device library. And if you find any bug, please let us know about it [reporting an issue](https://github.com/Creditas/vanilla-masker/issues).
 
-If you wanna see how this lib works, click to test this [demo page](http://bankfacil.github.io/vanilla-masker/demo.html).
```

Comment:
why did you remove the `demo.html` part? It seems to be working from [http://creditas.github.io/vanilla-masker/demo.html](http://creditas.github.io/vanilla-masker/demo.html) even it this page could be improved 😛